packages/http/README.md

```diff
@@ -110,24 +108,22 @@ const Prism = require('@stoplight/prism-http');
 const prism = Prism.createInstance({ mock: { dynamic: false } });
 
 // Make a "GET /todos" request
-return prism
-  .process(
-    {
-      method: 'get',
-      url: {
-        path: '/facts',
-        baseUrl: 'https://cat-fact.herokuapp.com',
-      },
+return prism.process(
+  {
+    method: 'get',
+    url: {
+      path: '/facts',
+      baseUrl: 'https://cat-fact.herokuapp.com',
     },
-    {
-      // We can override the default behaviour per request.
-      mock: {
-        dynamic: true,
-      },
+  },
+  {
+    // We can override the default behaviour per request.
+    mock: {
+      dynamic: true,
     },
-    operations
-  )
-  .then(prismResponse => console.log(prismResponse.output));
+  },
+  operations
+);
```

Comment:
```suggestion
)();
```
.. and `TaskEither.fold` should be there?

Reply:
Yes, I'll update the docs.